### DISCUSSION TITLE
how to send image file to api

### DISCUSSION THREAD

**Discussion by nixon42** [flutterdata/flutter_data#183] [Q&A]
sorry i am beginner to flutter, but how can i send image to api backend ?
i use django rest framework as back end,
thank you

**Comment by frank06** [comment]
Using Flutter or Flutter Data? What have you tried so far?

**Reply by nixon42** [reply]
with flutter_data, i know i can use multipartRequest from http lib to send file to API, so my current solution just save current model with flutter_data and then upload the image with http lib, i wonder is there better way to achieve this ?

thank you
